test/test_aqua.jl

```diff
@@ -1,8 +1,7 @@
-using Test
-using QuantumOptics
-
-@testset "Quality Assurance" begin
+@testitem "Quality Assurance" tags = [:aqua] begin
```

Comment:
In my initial comment, I mentioned just wrapping the existing testsets in a testitem. What you have done here is not exactly the same.

It would be easier to just NOT modify any of the content of the files except to add `@testitem ... begin` at the very start of the file, and `end` at the end of the file, and to not make any other changes (including not changing indentation). It would also lead to way fewer changes to the files as a whole (and should avoid the need to redefine some variables as globals)

Do not remove various `using` statements either, just keep them below the `@testitem`

Reply:
Sure, I will just add `@testitem`